tox.ini

```diff
@@ -49,3 +49,21 @@ commands = mkdocs build
 deps =
        -rrequirements/requirements-testing.txt
        -rrequirements/requirements-documentation.txt
+
+[testenv:py36-django32]
+ignore_errors = true
+
+[testenv:py37-django32]
+ignore_errors = true
+
+[testenv:py38-django32]
```

Comment:
Should we be allowing test failures on 3.2? It's due this week?

Reply:
Just copying what's there already - I think there's a process to update things when the release comes out.